kcp/manifests/plnsvc/namespace.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Lines 5,7,8,13,15,16 are leftover metadata I guess. Can be removed.

Reply:
Fixed